### PR TITLE
update icon: webstorm (original, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *update icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)

**This PR contributes to #1125**

## Link to prove your SVG is correct and up-to-date.

https://www.jetbrains.com/fr-fr/webstorm/

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient

File: icons/webstorm/webstorm-plain-wordmark.svg [hunk 1]
```diff
@@ -1 +1,3 @@
...
...0-.312.285-.55.738-.55a2.174 2.174 0 0 1 1.121.366l.258-.426a2.57 2.57 0 0 0-1.367-.418zm-10.035.007c-.056-.001-.112 0-.168.004-.481 0-.953.11-1.383.32l.172.473a2.609 2.609 0 0 1 1.152-.277c.71 0 1.133.351 1.133 1.035v.133a4.054 4.054 0 0 0-1.168-.164c-.966 0-1.625.43-1.625 1.23v.012c0 .782.71 1.184 1.426 1.172a1.666 1.666 0 0 0 1.375-.653v.575h.535v-2.325a1.487 1.487 0 0 0-.39-1.109 1.663 1.663 0 0 0-1.06-.426zm-2.34.004a1.613 1.613 0 0 0-.942.281 1.61 1.61 0 0 0-.61.778v-.992h-.566v3.8h.567v-1.515c0-1.145.676-1.742 1.508-1.742h.043v-.61zm-25.45.059 1.727 3.793c-.231.53-.441.71-.781.71a1.155 1.155 0 0 1-.594-.14l-.191.45a1.709 1.709 0 0 0 .808.187c.574 0 .942-.28 1.266-1.07l1.594-3.922h-.602l-1.223 3.16-1.382-3.168h-.622zm30.418.008v3.793h.563V72.64h-.563zm-17.847.394c.758 0 1.152.606 1.21 1.293h-2.445c.075-.738.57-1.293 1.235-1.293zm-14.871.031c.722 0 1.34.575 1.34 1.47v.015c0 .918-.606 1.46-1.34 1.46a1.42 1.42 0 0 1-1.375-1.468v-.016a1.423 1.423 0 0 1 1.375-1.46zm20.98 1.036h1.625c.9 0 1.367.328 1.363.882V75c0 .57-.484.918-1.261.918h-1.727v-1.816zm8.906.402a4.006 4.006 0 0 1 1.118.164v.375c0 .602-.576 1.03-1.282 1.02-.509 0-.949-.282-.949-.754v-.02c0-.477.403-.785 1.113-.785z"/>
```
Comment (Snailedlt) [inline icons/webstorm/webstorm-plain-wordmark.svg:2]: Missing fill?

Comment (kilian-paquier) [reply icons/webstorm/webstorm-plain-wordmark.svg:2]: Related to the #1291 thread

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Marking as request changes until #1291 is solved

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy_check_mark:

**Pull request merged**